python/MufluxDigiReco.py

```diff
@@ -151,6 +186,104 @@ def digitize(self):
         self.digitizeMufluxSpectrometer()
         self.digiMufluxSpectrometerBranch.Fill()
 
+        self.digiMuonTagger.Delete()
+        self.digitizeMuonTagger()
+        self.digiMuonTaggerBranch.Fill()
+
+
+
+    def digitizeMuonTagger(self):
```

Comment:
The indentation of this function is very confusing. Every few lines there are a few spaces more or less.